components/dataset/ViewCard.vue

```diff
@@ -0,0 +1,118 @@
+<script setup lang="ts">
+interface Props {
+  view: string;
+  tableName: string;
+}
+
+const props = defineProps<Props>();
+const { t } = useI18n();
+
+// Get view display name
+const viewDisplayName = computed(() => {
+  return t(props.view) || props.view;
+});
+
+// Get view description based on type
+const viewDescription = computed(() => {
+  switch (props.view) {
+    case "map":
+      return "Borders, production sites, demographic changes.";
+    case "gallery":
+      return "Showcase of animal species, habitats and related.";
+    case "alerts":
+      return "Wildfire, deforestation, extraction, violence rates.";
+    default:
+      return "";
+  }
+});
+
+// Determine which icon to show
```

Comment:
```suggestion
```

I won't suggest further removals of redundant function documentation in this review. But please adapt accordingly. (JSDoc for functions that requires them, remove any code documentation that is superfluous).